src/sentry/incidents/subscription_processor.py

```diff
@@ -708,43 +718,19 @@ def get_anomaly_data_from_seer(self, aggregation_value: float | None):
             )
         except (TimeoutError, MaxRetryError):
             logger.warning(
-                "Timeout error when hitting anomaly detection endpoint",
-                extra={
-                    "subscription_id": self.subscription.id,
-                    "dataset": self.subscription.snuba_query.dataset,
-                    "organization_id": self.subscription.project.organization.id,
-                    "project_id": self.subscription.project_id,
-                    "alert_rule_id": self.alert_rule.id,
-                },
+                "Timeout error when hitting anomaly detection endpoint", extra=extra_data
             )
             return None
 
-        if response.status != 200:
+        if response.status > 400:
             logger.error(
-                f"Received {response.status} when calling Seer endpoint {SEER_ANOMALY_DETECTION_ENDPOINT_URL}.",  # noqa
-                extra={"response_data": response.data},
+                "Error when hitting Seer detect anomalies endpoint",
+                extra=extra_data,
             )
             return None
-
         try:
-            results = json.loads(response.data.decode("utf-8")).get("timeseries")
-            if not results:
-                logger.warning(
-                    "Seer anomaly detection response returned no potential anomalies",
-                    extra={
-                        "ad_config": anomaly_detection_config,
-                        "context": context,
-                        "response_data": response.data,
-                        "reponse_code": response.status,
-                    },
-                )
-                return None
-            return results
-        except (
-            AttributeError,
-            UnicodeError,
-            JSONDecodeError,
-        ):
+            decoded_data = response.data.decode("utf-8")
```

Comment:
This breaks apart `json.loads(response.data.decode("utf-8"))` into 2 steps because I was seeing errors sometimes where the response was either `None` or a string or something